src/reporters/base-reporter.js

```diff
@@ -170,6 +176,14 @@ export default class BaseReporter {
     }
   }
 
+  convertBytes(bytes: number): string {
+    return humanizeBytes(bytes);
```

Comment:
Is this indirection really needed?

Reply:
I think it's nice to be able to swap libs as needed. If any other parts of the codebase decide to use this utility, swapping the external libs would become easier this way. To your point, though, I'm not sure it will ever be used outside of this context.

Reply:
I think swapping out libraries can still be done if needed by aliasing `humanizeBytes` to something else.